### PR TITLE
fix: websocket: close 1006 (abnormal closure): unexpected EOF

### PR DESCRIPTION
**Pull request by fabiooshiro**

### CODE REVIEW

File: internal/inputter/inputter.go [hunk 1]
```diff
@@ -91,27 +86,85 @@ func (w InputterWorker) watchNewInputs(
...
+		// new ones. There is a race condition where we might lose inputs sent between the
+		// readPastInputs call and the watchNewInputs call. Given that nonodo is a development node,
+		// we accept this race condition.
+		err = w.readPastInputs(ctx, client, inputBox, currentBlock)
+		if err != nil {
+			if retryCount > maxRetry {
+				return fmt.Errorf("readPastInputs: %w", err)
+			} else {
+				slog.Warn("Inputter", "error", err)
+				slog.Info("Inputter reconnecting", "reconnectDelay", reconnectDelay)
```
Comment (sandhilt) [inline internal/inputter/inputter.go:107]: Why need this? I'd like know why this change in behaviour?

Comment (fabiooshiro) [reply internal/inputter/inputter.go:107]: https://github.com/Calindra/nonodo/issues/80

**Review by sandhilt — APPROVED** [review]
LGTM!

File: internal/inputter/inputter.go [hunk 2]
```diff
@@ -91,27 +86,85 @@ func (w InputterWorker) watchNewInputs(
...
+
+		if ctx.Err() != nil {
+			return ctx.Err()
+		}
+
+		if err == nil {
+			return nil
+		}
+
+		slog.Warn("Inputter", "error", err)
```
Comment (DiegoSena) [inline internal/inputter/inputter.go:165]: It should be slog.Error, right ?

Comment (fabiooshiro) [reply internal/inputter/inputter.go:165]: hummm, good
